app.R

```diff
@@ -39,7 +39,12 @@ setnames(ogDT, colnames(ogDT), c("Year", "Geography", "Education", "Age", "Sex",
                                  "Multiple visible minorities 2", "Not a visible minority 2"))
 
 # Read file for scatter plot
-degInc <- read.csv("combined214and275.csv", check.names = FALSE)
+# Important to set check.names TRUE here. There is something about the column
```

Comment:
I noticed that your latest commit has started using the synthetic dataset, so this fix is no longer necessary.

In the future, it's a really really good idea to use something like `janitor::clean_names()`. Column names with spaces are annoying to work with.

Reply:
@DennisH3 What I might propose is to accept this change but comment it out in your synthetic version.

That way you have the code for the future and you can still move forward with the synthetic data.